### ISSUE TITLE
[api-documenter] Missing empty line after emitted HTML tags for tables

### ISSUE THREAD

**Issue by catloversg** [microsoft/rushstack#4878]
## Summary

In #4578, we emit HTML tags for tables instead of Markdown code, but the emitted tags make some Markdown code rendered incorrectly in some cases.

## Repro steps

- Clone https://github.com/catloversg/api-documenter-bug-pr-4578.
- Run `npm run doc`.
- Check `markdown\library.library.testfunction.md`.

**Expected result:**
There is a empty line between `</tbody></table>` and `**Returns:**`.

![expected result](https://github.com/user-attachments/assets/269bf3c5-184a-4f5c-ae56-a7f2ffccf5fa)

**Actual result:**
No empty line between `</tbody></table>` and `**Returns:**`.

![actual result](https://github.com/user-attachments/assets/6dc6a87e-e1c2-4aaa-9175-c03f75b7e4e4)

## Details

This bug was caused by #4578.

## Standard questions

Please answer these questions to help us investigate your issue more quickly:

| Question | Answer |
| -------- | -------- |
| `@microsoft/api-documenter` version? | 7.25.10 |
| Operating system? | Windows |
| Documentation target? | Markdown |
| Would you consider contributing a PR? | No |
| TypeScript compiler version? | 5.4.2 |
| Node.js version (`node -v`)? | 20.11.1 |

**Comment by octogonz** [comment]
Agree, according to CommonMark there should be a newline around HTML elements
